Please update custom_resources.rst

benvdh opened the pull request:
Hi Max,

I added a note and example code to clarify how to deal with a single custom resource. You need to add a trailing comma there, else python will see it as a string and iterate over every character in the string. This turns every character into a resource path!

maxtepkeev commented:
Hi @benvdh 

I really appreciate your feedback and desire to make python-redmine better, but I don't think that this is something that should be added to documentation, because this has nothing to do with python-redmine, but with Python itself:

```python
foo = ('foo.bar',)  # this is how you define a tuple in python
foo = ['foo.bar']   # you can also use a list, this way you don't need a trailing comma
```

What I'm trying to say is that python-redmine's documentation is not the right place to teach people how to write proper Python, there is a Python documentation for that, this is why I have to decline this pull request.

But once again, I really appreciate your feedback. Thanks.

benvdh commented:
Hi Max, Thank you for your feedback. I'll use the list way then. Thanks for the tip. I added some other points on the forums with regard to another method to query issues in more extensive way. I implemented it through the custom_resources feature now. Kind regards, Ben 12.08.2016, 18:00, "Max Tepkeev" <notifications@github.com>:Hi @benvdhI really appreciate your feedback and desire to make python-redmine better, but I don't think that this is something that should be added to documentation, because this has nothing to do with python-redmine, but with Python itself:foo = ('foo.bar',)  # this is how you define a tuple in python
foo = ['foo.bar']   # you can also use a list, this way you don't need a trailing commaWhat I'm trying to say is that python-redmine's documentation is not the right place to teach people how to write proper Python, there is a Python documentation for that, this is why I have to decline this pull request.But once again, I really appreciate your feedback. Thanks.—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread. 
